apps/ltn/src/per_resident_impact.rs

```diff
@@ -0,0 +1,395 @@
+                .build_def(ctx)
+                .centered_vert(),
+            Widget::vertical_separator(ctx),
+        ];
+        if self.current_target.is_none() {
+            row.push(
+                "Click a building outside the neighbourhood to see driving times there"
+                    .text_widget(ctx)
+                    .centered_vert(),
+            );
+        } else {
+            row.extend(vec![
+                "The time to drive from the neighbourhood to this destination changes:"
+                    .text_widget(ctx)
+                    .centered_vert(),
+                ColorLegend::gradient(
+                    ctx,
+                    &scale,
+                    vec!["0", &max_change.to_string(&UnitFmt::metric())],
+                )
+                .centered_vert(),
+                ColorLegend::row(ctx, *colors::PLAN_ROUTE_BEFORE, "before changes"),
+                ColorLegend::row(ctx, *colors::PLAN_ROUTE_AFTER, "after changes"),
+            ]);
+        }
+        self.bottom_panel = BottomPanel::new(ctx, &self.appwide_panel, Widget::row(row));
+
+        let map = &app.per_map.map;
+        self.world = World::bounded(map.get_bounds());
+
+        for b in map.all_buildings() {
+            if let Some((before, after)) = self.times_from_building.get(&b.id) {
+                let color = scale.eval((*after - *before) / max_change);
+                let mut txt = Text::from(if before == after {
+                    format!("No change -- {before}")
+                } else {
+                    format!(
+                        "{} slower -- {before} before this proposal, {after} after",
+                        *after - *before
+                    )
+                });
+                if before != after {
+                    txt.add_line(Line("Click").fg(ctx.style().text_hotkey_color));
+                    txt.append(Line(" to investigate"));
+                }
+
+                self.world
+                    .add(Obj::Building(b.id))
+                    .hitbox(b.polygon.clone())
+                    .draw_color(color)
+                    .hover_color(colors::HOVER)
+                    .tooltip(txt)
+                    .clickable()
+                    .build(ctx);
+            } else {
+                // TODO We run out of GPU memory on large maps if we hover here. Make World be
+                // lazier!
+                self.world
+                    .add(Obj::Building(b.id))
+                    .hitbox(b.polygon.clone())
+                    .drawn_in_master_batch()
+                    .invisibly_hoverable()
+                    //.hover_color(colors::HOVER)
+                    .clickable()
+                    .build(ctx);
+            }
+        }
+        self.world.initialize_hover(ctx);
+
+        if let Some(b) = self.current_target {
+            self.world.draw_master_batch(
+                ctx,
+                GeomBatch::load_svg(ctx, "system/assets/tools/star.svg")
```

Comment:
We more generally need a "louder" way to show this:
![Screenshot from 2023-02-02 15-06-24](https://user-images.githubusercontent.com/1664407/216361585-dabe7644-1308-4ac6-93e7-ae529ff87cda.png)